docs/AppFramework.md

```diff
@@ -64,34 +70,38 @@ spec:
 
 6. Apply the Custom Resource specification: `kubectl apply -f Standalone.yaml`
 
-The App Framework detects the Splunk App archive files available in the App Source locations, and deploys the apps to the standalone instance for local use. The App Framework will also scan for changes to the App Source folders based on the polling interval, and deploy updated archives to the instance. A Pod reset is triggered to install the new or modified apps.
+The App Framework detects the Splunk app or add-on archive files available in the App Source locations, and deploys them to the standalone instance path for local use. 
 
-Note: A similar approach can be used for installing apps on License Manager using it's own CR.
+The App Framework calculates a checksum for each app or add-on archive file in the App Source location. The app name and checksum is recorded in the CR, and used to compare the deployed apps to the app archive files in the App Source location. The App Framework will scan for changes to the App Source folders using the polling interval, and deploy any updated apps to the instance. 
```

Comment:
"The App Framework calculates a checksum for each app or add-on archive file in the App Source location". --> 
"The App Framework learns from remote store, and maintaines checksum for each app or add-on archive file in the App Source location"